src/Education/XamlGenerator/DiffTranslator.cs

```diff
@@ -72,7 +72,22 @@ private string HighlightLines(List<DiffPiece> lines, StyleResourceNames style)
                 {
                     writer.WriteStartElement("Span");
```

Comment:
foreach line that is different highlight the whole line with the normal style then wherever applicable, add a different style to individual sub pieces of the line